yourgov/components/FormMobileFoodVendorPermit/UploadFileTable.tsx

```diff
@@ -0,0 +1,212 @@
+import { forwardRef, Fragment } from 'react';
+import { VisuallyHidden } from '@ag.ds-next/react/a11y';
+import { SkeletonBox, SkeletonText } from '@ag.ds-next/react/skeleton';
+import {
+	Table,
+	TableBody,
+	TableCell,
+	TableHead,
+	TableHeader,
+	TableHeaderProps,
+	TableWrapper,
+	TableRow,
+} from '@ag.ds-next/react/table';
+import { Text } from '@ag.ds-next/react/text';
+import { Button } from '@ag.ds-next/react/button';
+import { Stack } from '@ag.ds-next/react/stack';
+import {
+	AlertFilledIcon,
+	DeleteIcon,
+	UploadIcon,
+} from '@ag.ds-next/react/icon';
+import { Heading } from '@ag.ds-next/react/heading';
+import { Box } from '@ag.ds-next/react/box';
+import { StatusBadge } from '@ag.ds-next/react/status-badge';
+import { Document } from './steps/FormStep9';
+
+const descriptionId = 'data-table-description';
+
+type UploadFileTableProps = {
+	/** The id of the heading that describes the table. */
+	headingId?: string;
+	/** The id of the table. */
+	tableId?: string;
+	/** Open drawer. */
+	openDrawer: (document: Document) => void;
+	/** Remove an uploaded file. */
+	removeFile: (document: Document) => void;
+	/** Data for upload documents table. */
+	documents: Document[];
+};
+
+export const UploadFileTable = forwardRef<
+	HTMLTableElement,
+	UploadFileTableProps
+>(function UploadFileTable(
+	{ documents, headingId, openDrawer, removeFile, tableId },
+	ref
+) {
+	const error = false;
+	const loading = false;
+
+	if (error) {
+		return (
+			<Stack gap={1}>
+				<AlertFilledIcon color="error" size="lg" />
+				<Heading type="h2" fontSize="lg">
+					Failed to load
+				</Heading>
+				<Text>
+					There was an error loading the data. Please try refreshing the page to
+					try again.
+				</Text>
+			</Stack>
+		);
+	}
+
+	return (
+		<Stack gap={0.5}>
+			<TableWrapper>
+				<Table
+					aria-rowcount={10}
+					{...(headingId && {
+						'aria-labelledby': headingId,
+						'aria-describedby': descriptionId,
```

Comment:
This and `descriptionId` should be removed
```suggestion
						'aria-describedby': descriptionId,
```